lib/checkPullRequestLabels.js

```diff
@@ -193,12 +193,13 @@ module.exports.checkChangelogLabel = async function (context) {
   );
 
   const userName = pullRequest.user.login;
-  const changelogLabel = getChangelogLabelFromPullRequest(pullRequest);
+  const changelogLabel = await getChangelogLabelFromPullRequest(pullRequest);
   const hasChangelogLabel = Boolean(changelogLabel);
-
   // If the PR has a changelog label, no action is required.
   if (hasChangelogLabel) {
-    const changelogLabelIsValid = matchChangelogLabelWithRegex(changelogLabel);
+    const changelogLabelIsValid = await matchChangelogLabelWithRegex(
```

Comment:
Why is await added here?

Reply:
Just for testing purpose i added await here removing it .